EIPS/eip-4337.md

```diff
@@ -39,19 +39,18 @@ This proposal takes a different approach, avoiding any adjustments to the consen
 * **UserOperation** - a structure that describes a transaction to be sent on behalf of a user. To avoid confusion, it is not named "transaction".
   * Like a transaction, it contains "sender", "to", "calldata", "maxFeePerGas", "maxPriorityFee", "signature", "nonce"
   * unlike a transaction, it contains several other fields, described below
-  * also, the "nonce" and "signature" fields usage is not defined by the protocol, but by each account implementation
+  * also, the "signature" field usage is not defined by the protocol, but by each account implementation
 * **Sender** - the account contract sending a user operation.
 * **EntryPoint** - a singleton contract to execute bundles of UserOperations. Bundlers/Clients whitelist the supported entrypoint.
-* **Bundler** - a node (block builder) that bundles multiple UserOperations and create an EntryPoint.handleOps() transaction. Note that not all block-builders on the network are required to be bundlers
+* **Bundler** - A node (block builder) that can handle UserOperations and create an EntryPoint.handleOps() transaction and add it to the next block. Note that not all block-builders on the network are required to be bundlers.
```

Comment:
And should we also elaborate a bit on how it could achieve that? Be a block builder, work with block-building infrastructure (e.g. mev-boost / PBS), or use `eth_sendRawTransactionConditional` where available.

Reply:
My point is that it should be clear that it is not a "standalone app that can send bundles at will"
(forget the technicalities: of course it can send it, but only because we're an ERC. it can't do anything useful (profitable), so it can only be used for testing)